src/components/card/flip-card.tsx

```
@@ -25,7 +54,24 @@ const FlipCard = ({ attached }: FlipCardParam) => {
+                  if (!url) return;
+                  return (
+                    <button
+                      onClick={() =>
+                        handleClick({ url, elementName: platform })
+                      }
+                      key={platform}
+                    >
+                      {platform}
+                    </button>
+                  );
+                })}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)